horizontal-profile/FISViewController.m

```diff
@@ -0,0 +1,195 @@
+//
+//  FISViewController.m
+//  horizontal-profile
+//
+//  Created by Tom OMalley on 3/2/15.
+//  Copyright (c) 2015 The Flatiron School. All rights reserved.
+//
+
+#import "FISViewController.h"
+
+@interface FISViewController ()
+@property (weak, nonatomic) IBOutlet UIImageView *coverImage;
+@property (weak, nonatomic) IBOutlet UIImageView *profileImage;
+@property (weak, nonatomic) IBOutlet UILabel *nameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *classNameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *interestingFactLabel;
+@property (weak, nonatomic) IBOutlet UITextView *myTextView;
+
+@property (strong, nonatomic) IBOutletCollection(UIImageView) NSArray *myImageViews;
+@property (strong, nonatomic) IBOutletCollection(UILabel) NSArray *myLabels;
+
+@property (strong, nonatomic) NSLayoutConstraint *coverImageHeightPortrait;
+@property (strong, nonatomic) NSLayoutConstraint *coverImageHeightLandscape;
+
+@property (strong, nonatomic) NSArray *textView6PlusLandscape;
+@property (strong, nonatomic) NSArray *textView6PlusPortrait;
+
+@end
+
+@implementation FISViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    // remove constraints
+    [self removeAllConstraints];
+    
+    // setup images
+    [self.coverImage setImage: [UIImage imageNamed:@"stuffmomnevertoldyou.jpg"]];
+    [self.profileImage setImage: [UIImage imageNamed:@"Blake_Farenthold.jpg"]];
+    
+    // set constraints
+    [self setPortraitConstraints];
+}
+
+- (void)didReceiveMemoryWarning
+{
+    [super didReceiveMemoryWarning];
+}
+
+-(void)viewDidLayoutSubviews
+{
+    [super viewDidLayoutSubviews];
+}
+
+-(void)willTransitionToTraitCollection:(UITraitCollection *)newCollection withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
+    //         if UITraitNameVerticalSizeClass = Compact >>> LANDSCAPE
```

Comment:
its supposed to be a chart explaining what the `UITraitName____SizeClass` stuff equates to